tests/next_tests/unit_tests/program_processor_tests/runners_tests/dace_tests/test_dace.py

```diff
@@ -17,17 +17,65 @@
 
 import gt4py._core.definitions as core_defs
 import gt4py.next as gtx
+import gt4py.next.common as gtx_common
 from gt4py.next.ffront.fbuiltins import where
 
 from next_tests.integration_tests import cases
-from next_tests.integration_tests.cases import E2V, cartesian_case, unstructured_case  # noqa: F401
+from next_tests.integration_tests.cases import E2V
 from next_tests.integration_tests.feature_tests.ffront_tests.ffront_test_utils import (
-    exec_alloc_descriptor,  # noqa: F401
+    Edge,
+    IDim,
+    Vertex,
     mesh_descriptor,  # noqa: F401
 )
 
 dace = pytest.importorskip("dace")
 
+from gt4py.next.program_processors.runners import dace as dace_backends
+
+
+@pytest.fixture(
+    params=[
+        pytest.param(dace_backends.run_dace_cpu_cached, marks=pytest.mark.requires_dace),
+        pytest.param(
+            dace_backends.run_dace_gpu_cached,
+            marks=(pytest.mark.requires_gpu, pytest.mark.requires_dace),
+        ),
+    ]
+)
+def cached_dace_backend(request):
+    """
+    Test fixture to select the dace backends only. In order to trigger `fast_call`,
+    we have to use the cached backend so that the same `CompiledSDFG` object is
+    used on the second SDFG call.
+    """
+    yield request.param
+
+
+@pytest.fixture
+def cartesian_case(request, cached_dace_backend):
+    yield cases.Case(
+        backend=cached_dace_backend,
+        offset_provider={"Ioff": IDim},
+        default_sizes={IDim: 10},
+        grid_type=gtx_common.GridType.CARTESIAN,
+        allocator=cached_dace_backend.allocator,
+    )
+
+
+@pytest.fixture
+def unstructured_case(request, cached_dace_backend, mesh_descriptor):  # noqa: F811
```

Comment:
```suggestion
def unstructured_case(request, cached_dace_backend, mesh_descriptor):  # noqa: F811 [redefined-while-unused]
```
According to the code style you should also add this short descriptive name.
But why is it actually needed?

Reply:
You are right, it is not needed.